rl_games/configs/atari/ppo_breakout_torch_multigpu.yaml

```diff
@@ -0,0 +1,84 @@
+params:  
```

Comment:
btw could you use breakout from envpool?

Reply:
Envpool is actually more tricky. When prototyping with envpool it's actually slower with multi-GPU, at least out of the box. This is because envpool uses different threads and have complex interactions with these threads that are a bit difficult to control. For this reason, I have chosen the regular gym API for controlled performance.